app/src/main/java/com/sygic/travel/sdkdemo/detail/PlaceDetailActivity.kt

```diff
@@ -21,12 +18,37 @@ import com.sygic.travel.sdkdemo.gallery.GalleryActivity
 import com.sygic.travel.sdkdemo.utils.Utils
 
 class PlaceDetailActivity : AppCompatActivity() {
+    private val stSdk = StSDK.getInstance()
 
-    private var id: String? = null
     private var views: Views? = null
+    private var id: String? = null
     private var ratingPattern: String? = null
     private var tagPadding: Int = 0
 
+    private val favoriteAddRemoveCallback = object : Callback<String?>() {
+        override fun onSuccess(data: String?) {}
+
+        override fun onFailure(t: Throwable) {
+	        runOnUiThread { Toast.makeText(this@PlaceDetailActivity, t.message, Toast.LENGTH_LONG).show() }
+	        t.printStackTrace()
+        }
+    }
+
+    private val loadAllFavoritesIdsCallback = object : Callback<List<String>?>() {
+        override fun onSuccess(data: List<String>?) {
+	        checkIfPlaceInFavorites(data)
+            setOnFavoriteChangeListener()
+        }
+
+        override fun onFailure(t: Throwable) {
+	        setOnFavoriteChangeListener()
+        }
+    }
+
+	private fun checkIfPlaceInFavorites(data: List<String>?) {
```

Comment:
bad method name. This does not check anything, it sets some property, or inits that property.